Lib/fontTools/otlLib/builder.py

```diff
@@ -20,7 +20,7 @@ def buildCoverage(glyphs, glyphMap):
 LOOKUP_FLAG_USE_MARK_FILTERING_SET = 0x0010
 
 
-def buildLookup(subtables, flags=0, markFilterSet=None):
+def buildLookup(subtables, flags=0, markFilterSet=None, optimize=None, builder=None):
```

Comment:
Hmm, I’m not really sure it is a good idea (layering wise) to pass feaLib builder to otlLib builder to use it, if anything it should be feaLib using otlLib API not the other way around.

Reply:
I did think about this, but then it seemed like a much bigger change. feaLib.builder currently contains several general classes for building OT lookup objects, and those classes actually have very little to do with FEA - they store a location attribute but that's it. Arguably they should be moved to otlLib where they would be available for other code which builds OT lookups - people writing their own feature compilers, and of course, things like this optimizer.

How does that idea sound? @brawer?

Reply:
> Arguably they should be moved to otlLib where they would be available for other code which builds OT lookups

indeed, that was the plan. We should complete that.

Reply:
OK. I'm going to do *that*, then I'm going to do #1991, then I'm going to do this.